content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -365,6 +365,43 @@ This applies only to significant, user-visible behaviors which impact the
 correctness of applications running on Kubernetes or that impact the
 administration of Kubernetes clusters, and which are being removed entirely.
 
+An exception to the above rule is **feature-gates**. Feature gates are key=value 
+pairs that allow for users to enable/disable experimental features.
+
+Feature gates are intended to cover the development life cycle of a feature - they 
+are not intended to be long-term APIs. As such, they are expected to be deprecated 
+and removed after a feature becomes GA or is dropped. 
+
+As a feature moves through the stages, the associated feature-gate will evolve 
```

Comment:
The "at the same time as announcing its deprecation" clause doesn't parse for me.  Can we just remove those words?